hudi-spark-datasource/hudi-spark/pom.xml

```diff
@@ -202,6 +202,12 @@
       <groupId>org.apache.hudi</groupId>
       <artifactId>hudi-common</artifactId>
       <version>${project.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.hive</groupId>
+          <artifactId>hive-storage-api</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
I don't think you need to do this exclusion. You are already explicitly defining this dependency in this POM fixed to a specific version.

Reply:
It seems these changes was made by @xushiyan, not sure why this exclusion was done if the dep is defined later on a test dependency. original spark 3.2 default profile pr https://github.com/apache/hudi/pull/4752/files#diff-626ac74908a8f50208219cea0d3774937c7df7cf7fca47b43df8d1acc01d71e9